src/main/resources/story/web_app/Google Search.story

```diff
@@ -7,8 +7,8 @@ Examples:
 |search-button      |By.xpath((//*[@name='btnK'])[1])|
 
 Scenario: Verify VIVIDUS documentation is in Top 10 Google search results
-Given I am on a page with the URL 'https://www.google.com/'
+Given I am on page with URL `https://www.google.com/`
 When I enter `VIVIDUS test automation` in field located `<search-input-field>`
-When I wait until element located `<search-button>` appears
+When I wait until element located by `<search-button>` appears
 When I click on element located `<search-button>`
```

Comment:
would you mind updating all steps to their non-deprecated alternatives?

Reply:
All deprecated steps are updated

Reply:
https://github.com/vividus-framework/vividus/blob/master/vividus-plugin-web-app/src/main/java/org/vividus/steps/ui/web/ElementSteps.java#L294-L295

Reply:
It's correct and as I see in my changes
![image](https://user-images.githubusercontent.com/30229905/218239337-e97fc3ab-f7b7-4e5b-8828-73aef71a4d6e.png)